src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -889,6 +896,7 @@ public static string GetAutoCompleteText(int index, string input, string query)
             {
                 if (index == 0)
                 {
+                    // Using OrdinalIgnoreCase since this is internal
```

Comment:
This is ordinal not because it's internal but because we want the characters to be exact in autocomplete text and the query.